hw/ip_templates/rstmgr/rstmgr_reg.core.tpl

```diff
@@ -10,6 +10,7 @@ virtual:
 filesets:
   files_rtl:
     depend:
+      - lowrisc:prim:subreg
       - lowrisc:tlul:headers
```

Comment:
Funny this depends only on tlul:headers and clkmgr on ip:tlul. It seems it should only one of the two uniformly.

Reply:
Since the reg_top is here, it should be ip:tlul. Nice catch!

Relatedly, I think I ought to move rstmgr_reg_pkg.sv over to the rstmgr_pkg core, so it can avoid pulling in RTL cores that can create dependency cycles. (similar to what has been done for the pwrmgr_pkg core)